src/routes/+page.svelte

```diff
@@ -11,4 +14,16 @@
     <label for="name">Project Name:</label>
     <input type="text" bind:value={name} class="border" />
   </div>
+
+  <!-- eslint-disable-next-line svelte/valid-compile -->
+  {#if $page.data.session?.user}
```

Comment:
It is reported in an open issue sveltejs/eslint-plugin-svelte#652 but has not been responded to yet.